content/docs/tech-insights/data-sources/index.md

```diff
@@ -0,0 +1,71 @@
+
+To manage Data Sources, go to Tech Insights → Data Sources.
+
+### Using Built-in Data Sources
+
+By default, Roadie includes 5 Data Sources for common vendors such as Datadog, Snyk, and GitHub. All built-in Data Sources have pre-defined Facts you can use for defining your [Checks](../checks//index.md).
+
+To use built-in Data Sources, you don’t need to do any set up. Jump to the next section to learn more about Checks.
+
+### Defining your own Data Source
+
+Roadie Tech Insights lets you define arbitrary Data Sources. Typically, custom Data Sources rely on calling an API or reading a configuration file in a repository. When adding a Data Source, you’ll also specify which Facts are extracted from said source.
+
+To add a Data Source, click on the “Add Data Source” button in the Data Sources page.
+
+![Add Data Source](./data-sources-overview.png)
+
+To set up a Data Source, you’ll be prompted to define a Data Provider and define which fields will be extracted as Facts from the data. Let’s start overviewing the options available for setting up a Data Provider:
+
+![Add Data Provider](./data-provider-step.png)
+
+1 - You must specify a type for that new Data Source. The _HTTP_ type lets you connect to an external API to pull in data, while the _Component repository file_ type lets you extract data from a file path in the corresponding repository of a given Component in your Catalog. As an example, let’s use “_Component repository file”_.
+
+2 - Set the path to the file you want to extract data from in repositories, starting from the root. This can be anything from JSON files to YAML files.
+
+3 - Try out what would be the response when testing specific entity from the location you have provided. If you were to get the `package.json` from a `sample-service` component, the Data Source would get something like this:
+
+![Dry run](./dry-run-result.png)
+
+Now that you have data, let’s define what Facts interest you. You’ll do this through the Fields Extraction section.
+
+![Fields extraction](./field-extraction.png)
+
+4 - Choose a parser to extract a Fact from the data obtained before. For the type “Component repository file” this can be either JSON or Regex parser type, while for “HTTP” data provider type, only JSON is supported. Keep in mind, JSON type may be used only for JSON type of files.
+
+JSON type of parser uses [JSONPath Plus](https://jsonpath-plus.github.io/JSONPath/docs/ts/) syntax to extract data from JSON, while Regex type uses [ECMAScript syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) to extract data from text.
```

Comment:
Love that we link to these docs. Is there a ticket to provide these links (with `target=_blank`) in the UI also?

Reply:
Not that I know, will check.